webui/react/playwright.config.ts

```diff
@@ -103,5 +99,6 @@ export default defineConfig({
     reuseExistingServer: !process.env.CI,
   },
 
-  workers: process.env.CI ? 4 : 1,
+  // workers: process.env.CI ? 4 : 1,
```

Comment:
can this get a ticket and/or explanation in the comment for future people?(I know we discussed this already)

Reply:
yup. it's in the description of the pr